block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagStorage.scala

```diff
@@ -32,6 +32,7 @@ trait BlockDagRepresentation[F[_]] {
   def lookupByDeployId(deployId: DeployId): F[Option[BlockHash]]
   def topoSort(startBlockNumber: Long): F[Vector[Vector[BlockHash]]]
   def topoSortTail(tailLength: Int): F[Vector[Vector[BlockHash]]]
+  def topoSortFragment(startBlockNumber: Long, endBlockNumber: Long): F[Vector[Vector[BlockHash]]]
```

Comment:
Can you please see if we can only have one function for `topoSort` which accepts start block and optionally end block.